class.jetpack-gutenberg.php

```diff
@@ -89,6 +94,57 @@ class Jetpack_Gutenberg {
 	 */
 	private static $availability = array();
 
+	/**
+	 * Check to see if a minimum version of Gutenberg is available. Because a Gutenberg version is not available in
+	 * php if the Gutenberg plugin is not installed, we also need the minimum WordPress version that was released
+	 * with the required Gutenberg version.
+	 *
+	 * @param array  $version_requirements An array containing the required Gutenberg version and the WordPress version that was released with this minimum version.
+	 * @param string $slug The slug of the block or plugin that has the gutenberg version requirement.
+	 *
+	 * @since 8.3.0
+	 *
+	 * @return boolean True if the version of gutenberg required by the block or plugin is available.
+	 */
+	public static function is_gutenberg_version_available( $version_requirements, $slug ) {
+		global $wp_version;
+
+		// Bail if the version requirements are not set correctly.
+		if ( empty( $version_requirements['gutenberg'] ) || empty( $version_requirements['wp'] ) ) {
```

Comment:
Should it be `&&`?

```suggestion
		if ( empty( $version_requirements['gutenberg'] ) && empty( $version_requirements['wp'] ) ) {
```

Reply:
haha, my fogging friday morning brain had to go away for ☕ and come back to this ... my thinking on this was that we need both the 'wp' and 'gutenberg' version details in order to do a reliable check on which gutenberg version we have, so if either of them (||) are not set we should bail. If we check if both of them (&&) are not set then we could proceed to do the checks if we had the 'gutenberg' version but not the 'wp' one.

Apologies if my caffeine infused brain is still confused here :-)

Reply:
...also, out of interest is there any sort of debug logging pattern in jetpack to indicate to a developer that this call is failing because they haven't pass in the correct params, eg. log a warning if WP_DEBUG is true?

Reply:
No you're right, that was me getting it wrong. This is good as it is! Sorry!